.gitpod.yml

```diff
@@ -17,7 +17,6 @@ tasks:
 
 ports:
   - port: 8080
-    onOpen: open-preview
```

Comment:
On purpose. If server loads while you are outside of Chrome window, and this preview opens, it crashes whole chrome. Happened multiple times, very annoying.